edx_sga/sga.py

```diff
@@ -913,6 +914,11 @@ def is_instructor(self):
         """
         Check if user role is instructor.
         """
+        # If it's a CCX course and the user has staff role (ccx_coach) or admin role (instructor)
+        # the user must be able to grade assignments without instructor approval.
+        if isinstance(self.course_id, CCXLocator):
+            return self.xmodule_runtime.get_user_role() in ('staff', 'instructor')
```

Comment:
I think it's neccesary add ccx_coach and data researcher roles following the implementation in course operations.

https://github.com/Pearson-Advance/course_operations/blob/master/pearson_course_operation/constants.py#L10

Reply:
The purpose of this PR is to migrate existing custom xblock features, new implementations need to be explicitly requested. @anfbermudezme